Lending.lir

```diff
@@ -0,0 +1,17 @@
+both(
```

Comment:
This case doesn't make much sense yet because of full collateralization. Let's remove it for now :-)

Reply:
sorry, actually, it was a mistake. Even if it would be included it should be in the examples folder

Reply:
np